### PR TITLE
Add 04-taipei sprint landing page

### PR DESCRIPTION
**Pull request by Li-Pro**
Reopen https://github.com/sciwork/swportal/pull/104

- Add 04-taipei page and updated some pelican metadata for sprint.
- These info need update:
  - [x] sprint location (& map)
  - [x] sprint note
  - [x] starting time (& duration)
  - [ ] additional projects

### CODE REVIEW

**Comment by yungyuc** [comment]
Thanks.  We should merge after the venue is booked and included in the page.  Let's wait for it.

File: content/pages/sprint/2022/04-taipei.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+|	Check out the `Fee`_, too.
+|
+
+.. Sponsors
+.. --------
+
+Venue
+-----
+
+`小樹屋｜香楠 703 <https://goo.gl/maps/8B2SaG4SynCK82es9>`__
```
Comment (yungyuc) [inline content/pages/sprint/2022/04-taipei.rst:51]: How about adding the link to the happ https://thehapp.com/space/336 and provide the google map separately.  The happ link provides some pictures that will be useful to attendees.

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: content/pages/sprint/2022/04-taipei.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+--------------------------------
+
+Add your name to the sprint note: https://hackmd.io/@sciwork/HJ_gzk-Gq .
+
+Date & time
+-----------
+
+9th April, Saturday, 2022, 14:00 -- 18:00
+
+Fee
```
Comment (yungyuc) [inline content/pages/sprint/2022/04-taipei.rst:31]: Fee is OK.  Alternate wording is cost sharing.

Comment (Li-Pro) [reply content/pages/sprint/2022/04-taipei.rst:31]: Cool, seems straightforward.

File: content/pages/sprint/2022/04-taipei.rst (hunk 1 above)
Comment (Li-Pro) [reply content/pages/sprint/2022/04-taipei.rst:51]: Ahh my bad. Forgot to put the official page 😂.

**Comment by Li-Pro** [comment]
@yungyuc thanks for reviewing. I added the requests in https://github.com/sciwork/swportal/pull/105/commits/932f4eff8fc8783ddc9cbca7e4c479d03f42e356 & https://github.com/sciwork/swportal/pull/105/commits/9d6717c13238a98694e0451e0e40c461a69ade23.

**Comment by Li-Pro** [comment]
Haha, there's a lot of commits XD.
Thought I should squash those commits before merging, but that's fine 😄.

**Comment by yungyuc** [comment]
Usually we will use a separate wip branch for intermediate checkins and squash to the under-review feature branch.  Rewriting the history of the under-review branch is annoying too.

**Comment by Li-Pro** [comment]
Oh, so it's like adding a draft branch for review, than pick & squash to another branch for PR right?

**Comment by yungyuc** [comment]
The PR (feature) branch is for review.  The other is for work in progress and not advertised to other people.  Code and changes on the WIP branch is squashed and merged to the feature branch after it is ready for review.

**Comment by yungyuc** [comment]
If it's only a couple of lines it doesn't really matter to tidy it or not.

If it's 500 lines of code, the contents and logic is more important than history rewriting.

If it's more than 500 lines of code in a PR, there's something seriously wrong.

**Pull request merged**